modules/javafx.graphics/src/main/java/com/sun/prism/es2/IOSGLDrawable.java

```diff
@@ -51,4 +52,8 @@ boolean swapBuffers(GLContext glCtx) {
         return nSwapBuffers(glCtx.getNativeCtxInfo(), getNativeDrawableInfo());
     }
 
+    @Override
+    void dispose() {
+        nDestroyDrawable(getNativeDrawableInfo());
```

Comment:
You should set `nativeDrawableInfo` to zero in this method (and check this before calling the native method), as otherwise we run the risk of freeing a stale pointer when the `dispose()` method is called repeatedly. The same should be done in all other implementations.

Reply:
You're right. done.